clientset/v1beta1/typed/v1beta1/internalversion/disruption.go

```diff
@@ -26,6 +26,7 @@ type DisruptionsGetter interface {
 // DisruptionInterface has methods to work with Disruption resources.
 type DisruptionInterface interface {
 	Create(ctx context.Context, disruption *v1beta1.Disruption, opts v1.CreateOptions) (*v1beta1.Disruption, error)
+	Update(ctx context.Context, disruption *v1beta1.Disruption, opts v1.UpdateOptions) (*v1beta1.Disruption, error)
```

Comment:
Do we need the update function? FWIK disruption are immutable. Ping @Devatoria @ptnapoleon

Reply:
Indeed 👍 Thanks for the reply 🙏